evap/evaluation/templates/base.html

```diff
@@ -85,49 +83,73 @@
             };
             activateTooltips();
 
-            // These are workarounds around incompatibilities of django, django-dynamic-formsets, and select2.
-            // select2 can't handle already select2'd forms that were copies made by
-            // django-dynamic-formsets' "add another" button, so we take a copy of a non-select2'd form here
-            var templateForm = $(".select2tr").last().clone();
+            // These are workarounds around incompatibilities of django, django-dynamic-formsets, and tomselect.
+            // tomselect can't handle already tomselect'd forms that were copies made by
+            // django-dynamic-formsets' "add another" button, so we take a copy of a non-tomselect'd form here
+            var templateForm = $(".tomselecttr").last().clone();
 
             // later we use this class to give this to django-dynamic-formsets
-            // as template and to make sure it does not get select2'd
+            // as template and to make sure it does not get tomselect'd
             templateForm.addClass("form-template").css("display", "none");
             // for some reason django-dynamic-formsets does not hide the checkbox like it should
             templateForm.find(':checkbox').last().attr("type", "hidden");
             // For some reason, django validates this template if it's part of the form, so we insert the copy outside of the form.
-            templateForm.insertAfter($('.select2form'));
+            templateForm.insertAfter($('.tomselectform'));
 
-            // put this into a function for later re-use
-            applySelect2 = function(elem, parent = $(document.body)) {
-                elem.each(function() {
-                    $(this).select2({
-                        language: "{{ LANGUAGE_CODE }}",
-                        placeholder: "{% trans 'Please select...' %}",
-                        minimumResultsForSearch: 15,
-                        minimumInputLength: $(this).children('option').length > 50 ? 3 : 0,
-                        allowClear: this.multiple,
-                        dropdownParent: parent
-                    });
-                });
+            // used in derived templates
+            const tagTomSelectOptions = {
+                create: true,
+                dropdownParent: document.createElement("div"),
```

Comment:
This reuses the same `div` for all users, I am not sure if this is a problem though :thinking:

Reply:
Doesn't matter, the div is not part of the document. The purpose is to hide the dropdown. Added a comment explaining that.